internal/pkg/agent/application/monitoring/liveness.go

```diff
@@ -56,6 +56,13 @@ func livenessHandler(coord CoordinatorState) func(http.ResponseWriter, *http.Req
 	return func(w http.ResponseWriter, r *http.Request) error {
 		w.Header().Set("Content-Type", "application/json; charset=utf-8")
 
+		if coord == nil {
```

Comment:
Is not having a coordinator the best way to detect that elastic-agent didn't complete enrollment yet ?
I was expecting something related to `managedConfigManager` and the existence of a fleet url and  token.

Reply:
If enrollment fails, we retry indefinitely.  However, if I'm following this code path correctly, `coord` will always be `nil` here in the container scenario so the liveness check will continue to succeed.  Is this the expected behavior?

Reply:
Yes we want it to report that it is alive. It is alive and trying to enroll.